webdriver/tests/actions/support/test_actions_wdspec.html

```diff
@@ -132,27 +145,31 @@
               -(areaRect.top + (event.clientY - boxRect.top)),
               20);
           els.dragArea.addEventListener("mousemove", moveHandler);
-          els.dragArea.addEventListener("mouseup", drop(moveHandler), {once: true});
+          initialMoveHandler = drop(moveHandler);
+          els.dragArea.addEventListener("mouseup", dropOnce);
```

Comment:
I'd like to avoid the global `initialMoveHandler`. Maybe define `dropOnce` inside `grab`, then you shouldn't need `initialMoveHandler`.

Reply:
I've gone as far as I can trying to eliminate the global event handler. If someone would show me the way to do that, I'd be happy to oblige.